test.sh

```diff
@@ -94,7 +94,9 @@ fi
 #
 if [[ "$RUN" =~ "integration" ]] ; then
   args=("--chisel")
-  args+=("--load")
+  if [[ "${INT_SKIP_LOAD}:-}" == "" ]]; then
```

Comment:
There is an extra `}` here after the `INT_SKIP_LOAD`.

Reply:
D'oh! Good catch, thanks! Fixed in f4550f3